jenkinsapi/build.py

```diff
@@ -91,6 +91,16 @@ def get_revision_branch(self):
         vcs = self._data['changeSet']['kind'] or 'git'
         return getattr(self, '_get_%s_rev_branch' % vcs, lambda: None)()
 
+    def get_params(self):
+        """
+        Return a dictionary of params names and their values or None
+        if no parameters present
+        """
+        actions = self._data.get('actions')
+        if actions:
+            parameters = actions[0].get('parameters', {})
```

Comment:
Ok, I see, order is not defined, need to iterate over the whole list to get parameters. Thanks, will make an update on that.